tests/integration/agents/kyma/test_kyma_agent_tool_accuracy.py

```diff
@@ -232,146 +232,123 @@ def create_test_cases_namespace_scoped(k8s_client: IK8sClient):
             ],
         ),
         TestCase(
-            "Should not call kyma doc search tool as Function is successfully deployed",
```

Comment:
please bring back this test case as if resource is read, no kyma doc search is needed.